src/PiggyCustomEnchants/Tasks/EffectTask.php

```diff
@@ -65,15 +65,15 @@ public function onRun(int $currentTick)
             if ($enchantment !== null) {
                 $effect = Effect::getEffect(Effect::STRENGTH);
                 $effect->setAmplifier($enchantment->getLevel() - 1);
-                $effect->setDuration(10);
+                $effect->setDuration(8);
                 $effect->setVisible(false);
                 $player->addEffect($effect);
             }
             $enchantment = $this->plugin->getEnchantment($player->getInventory()->getBoots(), CustomEnchants::GEARS);
             if ($enchantment !== null) {
                 $effect = Effect::getEffect(Effect::SPEED);
                 $effect->setAmplifier(0);
-                $effect->setDuration(10);
+                $effect->setDuration(4);
```

Comment:
🤦‍♂️ This isn't fixing... it's breaking.... The task runs every 5 ticks but the effect only lasts 4?